codechain/crypto/ecdsa.py

```diff
@@ -0,0 +1,123 @@
+from coincurve import PrivateKey
+from coincurve import PublicKey
+from coincurve import verify_signature
+
+SECPK1_N = (
+    115792089237316195423570985008687907852837564279074904382605163141518161494337
+)
+
+# Gets ECDSA signature for message from private key.
```

Comment:
Function/class documentations are inconsistent over the codebase. Would you document them with the docstring? https://www.python.org/dev/peps/pep-0257/

Reply:
I fixed it.